README.md

```diff
@@ -1,4 +1,12 @@
+# SHAP Community Fork
 
+This repository is a fork of Scott-Lundberg's popular [shap](https://github.com/slundberg/shap) library. Unfortunately, the original shap repo is not currently maintained, so this fork attempts to fix shap's current issues and merge old PRs. See the general discussion on the purpose of this repo [here](https://github.com/dsgibbons/shap/discussions/11).
```

Comment:
> Unfortunately, the original shap repo is not currently maintained, so this fork attempts to fix shap's current issues and merge old PRs

 Unfortunately, the original shap repo is not currently maintained. This fork attempts to fix shap's current issues and merge old PRs.